### DISCUSSION TITLE
would like to have write permission to trigger PR ci and request team to review

### DISCUSSION THREAD

**Discussion by trim21** [minio/minio-js#1233] [Q&A]
hi @harshavardhana, I plan to fully migrate this project to TypeScript, which need many PR. can I have a minimal permission to trigger PR ci and request team to review my PR, also merge them when try are approved, maybe?

**Comment by harshavardhana** [comment]
@trim21 due to compliance requirements it's not possible unfortunately. Unless you can join us in official capacity.

**Reply by trim21** [reply]
OK，thanks
